webroot/js/app.js

```diff
@@ -217,6 +217,66 @@ $(function() {
     // Attach filter to dropdown
     ddElement.insertBefore(textElement, ddElement.firstChild);
   }
-  
 
+  function setCookie(key, value, expiry) {
+    var expires = new Date();
+    expires.setTime(expires.getTime() + (expiry * 24 * 60 * 60 * 1000));
+    document.cookie = key + '=' + value + ';expires=' + expires.toUTCString();
+  }
+
+  function getCookie(key) {
+    var keyValue = document.cookie.match('(^|;) ?' + key + '=([^;]*)(;|$)');
+    return keyValue ? keyValue[2] : null;
+  }
+
+  function eraseCookie(key) {
+    var keyValue = getCookie(key);
+    setCookie(key, keyValue, '-1');
+  }
+
+  let sliderOpt = $('#day-night-slider > input');
+  if (sliderOpt.size() === 1) {
+      let slider = sliderOpt[0];
+      $(slider).change(function() {
+          if(this.checked) {
+            $('body').addClass("dark");
+            $('#day-night-slider > label > span')[0].innerHTML = "Night";
+            setCookie("lastChosenColorScheme", "dark", 30);
```

Comment:
So the cookies versus local storage thing was actually because I was debating making some of the other apps also integrate with the same dark selection...

Realistically, I ain't going to get to those for a while though, so it's probably fine to just use local storage for now.

Reply:
Ah .. that makes sense! .. maybe in the future we can have a separate service for user preferences across all DMS apps 😊

Reply:
Yeah, as I was kind of rereading this code just now and thinking about it- that'll probably make the most sense. There's some work that I've put into making a service that could handle that for DMS, so I guess whenever that happens I can revisit it.

I'll go ahead and get this using local storage instead for now later tonight